sdk/core/core-lro/src/http/operation.ts

```diff
@@ -316,7 +316,7 @@ export function getResourceLocation<TState>(
   state: RestorableOperationState<TState>
 ): string | undefined {
   const loc = accessBodyProperty(res, "resourceLocation");
-  if (loc !== undefined) {
+  if (loc) {
```

Comment:
I was just wondering if the loc is not a string, will the poller work correctly?

Reply:
Good point, I opened https://github.com/Azure/azure-sdk-for-js/pull/27875.